qa/os/src/test/java/org/elasticsearch/packaging/test/KeystoreManagementTests.java

```diff
@@ -319,6 +332,7 @@ public void test61DockerEnvironmentVariablePasswordFromFile() throws Exception {
      * Check that if we provide the wrong password for a mounted and password-protected
      * keystore, Elasticsearch doesn't start.
      */
+    @AwaitsFix(bugUrl = "Keystore fails to save with resource busy")
```

Comment:
This is not actually related.
It fails intermittently on me because there's not enough time for the docker run process to fail (it takes more than 5 seconds for the container to fail to start).

Reply:
Bumped it to 8 sec and we'll see